tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/design-view/form-builder/forms/stream-form.js

```diff
@@ -285,6 +285,9 @@ define(['require', 'log', 'jquery', 'lodash', 'attribute', 'jsonValidator', 'con
                     self.configurationData.setIsDesignViewContentChanged(true);
                     // close the form window
                     self.consoleListManager.removeFormConsole(formConsole);
```

Comment:
do we need to call removeFormConsole every time ? doesn't removeAllConsoles will close all consoles ?